VERSION.cmake

```diff
@@ -1,3 +1,3 @@
-SET(CR_MAJOR "0")
-SET(CR_MINOR "20")
-SET(CR_PATCH "1")
+SET(CR_MAJOR "1")
```

Comment:
We have discussed this in our team and we probably don't want to do this change just now even though we have stable API.
The change it self could confuse users especially since we don't do any breaking changes.
Can you please drop it? 
Otherwise it looks good to me.

Reply:
I'd note that the change in semver from 0.x to 1.x **is** arbitrary; it's whenever you think your API is mature enough to derlare it stable. If you're using semver and (as you've claimed) have a stable API you **must** have a major version of at least `1`.

I'm happy to revert the change, but, why do you think that your software declaring that it's mature enough to be v1.0.0 would scare users?

As an end-user myself, I'm far more scared of relying on a critical piece of software that (based on its versioning) is in _perpetual_ alpha.

Reply:
another note - if you don't set the major version to one here (which won't have any impact until you tag that release with whatever other changes you decide to include) you're **not** actually using semver. You're using "rpm-software-management's almost-but-not-quite semver" which will just add to the confusion going forward.

Reply:
Well no, it's using semver to declare a semver stability level of "whether we intend to break API or not, if we do end up doing so it's not a bug and you just have to deal with it".

:D

I do agree that a project committing to 1.0 is definitely not going to confuse users, ever.

Reply:
Specifically, according to https://semver.org

> 4. Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
> 
> 5. Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.

And:
> 8. Major version X (X.y.z | X > 0) MUST be incremented if any backwards incompatible changes are introduced to the public API. 

Notice that bumping the major version from 0 to 1 is rigidly defined as an exception to the normal rule that bumping the major version indicates backwards incompatible changes "to the public API".

Instead, bumping from 0 to 1 means that an unstable private API has now been marked as public. Rule 8 is only applicable to major versions >= 1

Reply:
I think I agree with you, I will try to push for 1.0.0.

Thank you for the inputs and the PR!